README.md

```diff
@@ -15,26 +15,26 @@ diagnostics. The package does not attempt to cover the full scope of
 HGVS recommendations. Please refer to
 [issues](https://github.com/biocommons/hgvs/issues) for limitations.
 
-### **Information**
+### Information
```

Comment:
Need to resolve header increments
```suggestion
## Information
```

Reply:
I have these as this size intentionally (all as being under the HGVS header) but I understand how this can be confusing. I'll make them the bigger size